designsafe/apps/api/notifications/tests.py

```diff
@@ -30,7 +30,7 @@
 
 # Create your tests here.
 @skip("Need to mock websocket call to redis")
-class NotificationsTestCase(TestCase):
+  class NotificationsTestCase(TestCase):
```

Comment:
Classes shouldn't be indented like this, I believe Josue was suggesting to keep two line breaks before the first class definition in a given file

```suggestion
class NotificationsTestCase(TestCase):
```